app/models/course_group.rb

```diff
@@ -4,6 +4,8 @@ class CourseGroup < ApplicationRecord
 
   validates :name, presence: { message: "Enter a unique course group name" }, uniqueness: { message: "Course name already exist, enter a unique name" }
 
+  scope :leadership_or_specialist, -> { where(name: %w[leadership specialist]) }
```

Comment:
Hi @mooktakim Could you please add a spec for this scope? I followed your suggestion on the other PR and added a spec in there.. so whatever PR gets merged first, the scope will be in.. tks
https://github.com/DFE-Digital/npq-registration/pull/1499/files#diff-be233ac6eae795dbfb4a1a24629eef68a767d6875377e03a5c7d1b7cc84c3dd6R80

Reply:
@leandroalemao i was planning on merging your ticket first as its almost ready to be merged. 👍

Reply:
I'll copy in the test anyway, it should be easy to merge.

Reply:
This has been added now in three prs, @leandroalemao maybe we can open a separate pr for this scope and merge so the others can rebase those changes?

Reply:
👍 I've got a PR to merge tomorrow which includes this changes .. so it should be good after that.